components/docker-up/docker-up/main.go

```diff
@@ -122,6 +123,14 @@ func runWithinNetns() (err error) {
 	}
 	args = append(args, userArgs...)
 
+	containerIf := "ceth0"
+	netIface, err := netlink.LinkByName(containerIf)
+	if err != nil {
+		return xerrors.Errorf("cannot get container network device %s: %w", containerIf, err)
+	}
+
+	args = append(args, fmt.Sprintf("--mtu=%v", netIface.Attrs().MTU))
+
```

Comment:
Is it worth while to run a perf test again, to see if we get similar to the results that @utam0k shared initially?

I ask because `netlink` is described as `Netlink is the interface a user-space program in linux uses to communicate with the kernel.`

Reply:
> Is it worth while to run a perf test again,

No. We are only referencing the interface we already have by name. No changes were introduced.